src/pdfpc.vala

```diff
@@ -122,8 +122,11 @@ namespace pdfpc {
                 ref Options.external_script,
                 "Enable execution of an external script", "file"},
             {"size", 'Z', 0, OptionArg.STRING,
-                ref Options.size,
-                "Size of the presentation window (implies \"-w\")", "W:H"},
+                ref Options.presentation_size,
+                "Size of the presentation window (implies \"-w both\")", "W:H"},
+            {"pc-size", 'A', 0, OptionArg.STRING,
```

Comment:
I'd probably use `-z` instead of `-A`. @AndreasBilke , what do you think?

Reply:
For my part, I just named it `-A` because the other PR did, and named it `--pc-size` because it was the shortest long-form I could come up with. I have strong feelings about neither.